infra/syntax/JsonFormatter.cpp

```diff
@@ -19,14 +20,30 @@ namespace infra
         {
             switch (c)
             {
-            case '"':  stream << "\\\""; break;
```

Comment:
We should rewrite this to have multiple fall-through case statements ending in "stream << "\\" << c;"